mod_reforged/hooks/config/character.nut

```diff
@@ -25,6 +25,9 @@ local getClone = ::Const.CharacterProperties.getClone;
 ::Const.CharacterProperties.OffensiveReachIgnore <- 0;
 ::Const.CharacterProperties.BonusPerReachAdvantage <- 0;
 
+::Const.CharacterProperties.IsImmuneIsOnToEnterPushback <- false;	// Will an enemy spearwall push me back to my original tile when they hit the AOO?
+::Const.CharacterProperties.IsSpearwallPushingBack <- false;		// Will my spearwall-like AOO push enemies back that I hit when they try to enter?
```

Comment:
A better name would be `IsPushingBackOnEnterAttack` or `IsPushingBackOnZOCEnterAttack` to go in line with the vanilla triggers for this attack i.e. `IsAttackingOnZoneOfControlEnter`.

Reply:
I like writing out the ZOC in this case.

How about `IsPushingBackOnZoneOfControlEnter`? So also without the `Attack` part.

Reply:
That would be misleading because this property only pushes back after a successful attack.